templates/containers/chiselled/index.html

```diff
@@ -66,7 +91,8 @@ <h2>Why your attack surface matters</h2>
     </div>
     <div class="col">
       <div class="p-section--shallow">
-        <p>Container images’ attack surface is a critical factor in determining their security. As the size of a container
+        <p>Container images’ attack surface is a critical factor in determining their security. As the size of a
```

Comment:
`’` is not a standard character, so it may not render well everywhere, it's usually `'`